docs/api/rest.md

```diff
@@ -655,6 +655,57 @@ Returns a list of states that have changed while the service was being executed.
 ]
 ```
 
+:::tip
+The result will include any states that changed while the service was being executed, even if their change was the result of something else happening in the system.
+:::
+
+If the service you're calling supports returning response data, you can retrieve it by adding `?return_response` to the URL. Your response will then contain both the list of changed entities and the service response data.
+
+```json
+{
+    "changed_states": [
+        {
+            "attributes": {},
+            "entity_id": "sun.sun",
+            "last_changed": "2024-04-22T20:45:54.418320-04:00",
+            "state": "below_horizon"
+        },
+        {
+            "attributes": {},
+            "entity_id": "process.Dropbox",
```

Comment:
```suggestion
            "entity_id": "binary_sensor.dropbox",
```

Reply:
All entities in the state machine should always be lower case

Reply:
That's copy-pasted from another example in this same document.

https://github.com/home-assistant/developers.home-assistant/blob/5d9b6a8a7658ed4dee96fd523f337ee16b7e4d17/docs/api/rest.md?plain=1#L390

https://github.com/home-assistant/developers.home-assistant/blob/5d9b6a8a7658ed4dee96fd523f337ee16b7e4d17/docs/api/rest.md?plain=1#L651

Are those wrong too? They look like very old examples, if the date in the sample data is to be believed.